app/views/admin/service_stages/unscoped_index.html.erb

```diff
@@ -0,0 +1,74 @@
+<% content_for :navigation_title do %>
```

Comment:
`/service_stages/index.html.erb` assumes a Service be passed into it, 
such that the Service Stages are shown for a specific Service.

`unscoped...` is about seeing all Service Stages, regardless of Service